front/lib/api/assistant/agent_usage.ts

```diff
@@ -133,6 +132,8 @@ export async function getAgentUsage(
 export async function agentMentionsCount(
   workspaceId: number
 ): Promise<mentionCount[]> {
+  const rankingTimeframeDays = (rankingTimeframeSec / 60) * 60 * 24;
```

Comment:
Why do the operation the other way there?
Maybe create a const USAGE_DAYS = 30 and use it in const rankingTimeframeSec = 60 * 60 * 24 * USAGE_DAYS ;

Reply:
Yeah, same right?